README.md

```diff
@@ -45,34 +57,52 @@ var connection = EdgeDBConnection.Parse("edgedb://user:password@localhost:5656/m
 var client = new EdgeDBClient(connection);
 ```
 
-### Querying the database
-```cs
+### Executing queries
 
-var result = await client.QueryAsync<long>("select 2 + 2"); // 4
-```
+**Note**: EdgeDB.Net is a fully asynchronous driver, and as such, all I/O
+operations are performed asynchronously.
 
-**Note**
-The `QueryAsync` method always returns a `IReadOnlyCollection<T>`, regardless of actual query cardinality. If you want to explicitly define cardinality when querying, you can use the `QuerySingleAsync` or `QueryRequiredSingle` methods.
+----
 
+Queries are executed through the `EdgeDBClient` by using different helper
+methods. Your choice of method is dependent on the kind of query you're making,
+better known as [cardinality](https://www.edgedb.com/docs/clients/dotnet/index#cardinality-and-return-types).
 
-**Note**
-EdgeDB.Net is a fully asynchronous driver, and as such, all IO operations are performed asynchronously.
+Query helper methods will expect a generic `T` type. It's recommended to use
+the [.NET version of an EdgeDB type](https://www.edgedb.com/docs/clients/dotnet/datatypes#datatypes),
+shown below:
+
+```cs
+var result = await client.QueryAsync<long>("select 2 + 2"); // returns 4
+```
+
+## Contributing
+
+We openly welcome and accept contributions to EdgeDB.Net! Before writing a
+GitHub Issue or Pull Request, please see our [contribution requirements](CONTRIBUTING.md).
 
 ## Examples
-You can view our curated examples [here](examples/EdgeDB.Examples.CSharp/Examples). We also have a demo asp.net project you can view [here](examples/EdgeDB.Examples.ExampleTODOApi). You're more than welcome to contribute to the examples!
+
+This repository contains a list of [working examples](examples/EdgeDB.Examples.CSharp/Examples),
+check them out to see EdgeDB.Net in action!
+
+We also have a [demo ASP.NET project](https://github.com/quinchs/EdgeDB.Net/tree/dev/examples/EdgeDB.Examples.ExampleTODOApi).
 
 ## Compiling
-If you want to build the EdgeDB.Net project from source, you will need the [.NET 6 SDK](https://dotnet.microsoft.com/en-us/download).
+
+If you're building EdgeDB.Net from its source, you will need to download the
+[.NET 6 SDK](https://dotnet.microsoft.com/en-us/download).
   
-Once you have the SDK installed you can run the dotnet build command in the root directory of the project:
+Once you have the SDK installed, you can then run `dotnet` in the root
```

Comment:
```suggestion
Once you have the SDK installed, you can then run `dotnet build` in the root
```

Reply:
I feel like having "`dotnet build`" with no extra args as bot a inline block and code block is repetitive (and breaks grices maximn) :/ but having the single command (dotnet) here for building and testing also doensn't feel right